### PR TITLE
Improve Login/Signup Experience

### LINKED ISSUE #226: Improve Login/Signup Experience
### Overview
Improve the design of login/signup pages.

Note: create-account and signup are the same thing

### Details
- See [this Figma](https://www.figma.com/file/ETUKCDm2THyAdUsRpWlPlM/Helping-Mamas-%2F-Spr23?node-id=2675%3A28708&t=CwnQr3QRR8IKVKVz-1) for how the login/signup page should look (the code for this is in AuthPage and AuthForm.jsx)
- Check to ensure the password and confirm password values are the same. If they are not, display an error to the user.

You shouldn't need to change any backend functionality for this ticket.

### PR DESCRIPTION
**Pull request by bphoebew**
Improving the design of login/signup pages

### CODE REVIEW

File: src/screens/Auth/helpers.js [hunk 1]
```diff
@@ -7,10 +8,23 @@ export const createAccountValidator = object().shape({
   password: string()
     .required("No password provided.")
     .min(10, "Password is too short - should be 10 chars minimum."),
+  password_confirm: string().required("Confirm your password."),
+
   // .matches(/[a-zA-Z]/, "Password can only contain Latin letters."),
   // yup-password
 });
 
+export const validationSchema = yup.object({
```
Comment (colin4554) [inline src/screens/Auth/helpers.js]: What is validationSchema for?

File: src/screens/Auth/helpers.js [hunk 2]
```diff
@@ -7,10 +8,23 @@ export const createAccountValidator = object().shape({
   password: string()
     .required("No password provided.")
     .min(10, "Password is too short - should be 10 chars minimum."),
+  password_confirm: string().required("Confirm your password."),
+
   // .matches(/[a-zA-Z]/, "Password can only contain Latin letters."),
```
Comment (colin4554) [inline src/screens/Auth/helpers.js]: Please remove any comments we don't need

**Review by colin4554 — CHANGES_REQUESTED** [review]

**Comment by colin4554** [comment]
@YiQin93 Can we remove/change the placeholder text in the fields? I'm not sure that the current placeholder text adds anything. It would probably make the most sense to either remove it or make it better (ie `jsmith@outlook.com` for the email placeholder)

**Comment by YiQin93** [comment]
> @YiQin93 Can we remove/change the placeholder text in the fields? I'm not sure that the current placeholder text adds anything. It would probably make the most sense to either remove it or make it better (ie `jsmith@outlook.com` for the email placeholder)

Yeah sure!! The current placeholder "Example@email.com" makes sense!!! Thanks for catching this~~ :)

**Comment by YiQin93** [comment]
This is very neat and user-friendly! It looks awesome!

There are two small issues:
1. Keeping the wording consistent could be very beneficial for users to digest information. For consistency, it would be wonderful if we could make the following changes:
- change the text pointed by the 1st arrow from "Create an account" to "Create an Account"
- change the text pointed by the 2nd arrow from "Sign up with Google" to "Continue with Google"
- change the text pointed by the 3rd arrow from "Sign up" to "Create an account"
- change the text pointed by the 4th arrow from "Sign in" to "Sign In"
(It would be even better if we could also pay attention to the capitalization!)
image

2. The error message for passwords with only numbers is somewhat confusing. Can we please change it to "Password should at least contain 1 letter"?
image

**Comment by YiQin93** [comment]
This looks great!!! Thanks Phoebe for fixing it right away! Their Login/Signup experience has been largely improved :)

**Review by colin4554 — APPROVED** [review]

**Pull request merged**